lib/lorekeeper/json_logger.rb

```diff
@@ -64,7 +63,12 @@ def remove_fields(fields)
     # @param exception: instance of a class inheriting from Exception
     # We will output backtrace twice. Once inside the stack so it can be parsed by software
     # And the other inside the message so it is readable to humans
-    def exception(exception, custom_message = nil, custom_data = nil)
+    def exception(exception, custom_message = nil, custom_data = nil, level = :error)
+      log_level = METHOD_SEVERITY_MAP[level]
+      unless log_level
+        return log_data(METHOD_SEVERITY_MAP[:warn], "Logger exception called with an invalid level: '#{level}'.")
+      end
```

Comment:
added this check to ensure if the user decides to log exception with another level doesn't shoot herself in the foot.
returning the `log_data` call because this is what happens further below.

Reply:
umm, but below, after the warning the data is still logged, here we lose the data. 
For this method, I'd only allow `level` to be `:error` or `:fatal`. You could just check that or just 
`log_level = METHOD_SEVERITY_MAP[level] || ERROR` and force a sane level that way.